### PR TITLE
Refactoroo

### PR DESCRIPTION
**Pull request by hedvigoscar**

### CODE REVIEW

File: owldroid/src/main/res/layout/fragment_charity.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    tools:context="com.hedvig.android.owldroid.ui.profile.charity.CharityFragment">
+                                                 xmlns:app="http://schemas.android.com/apk/res-auto"
```
Comment (FredrikAreschoug) [inline owldroid/src/main/res/layout/fragment_charity.xml]: We should sync so we have the same setting my editor will just push this back should it be like this?

File: owldroid/src/main/java/com/hedvig/android/owldroid/feature/marketing/ui/MarketingFragment.kt [hunk 2]
```diff
@@ -190,7 +192,8 @@ class MarketingFragment : Fragment() {
 
             blurOverlay.show()
             topHideAnimation = ValueAnimator.ofFloat(1f, 0f).apply {
-                duration = BLUR_ANIMATION_SHOW_DURATION
+                duration =
+                    BLUR_ANIMATION_SHOW_DURATION
```
Comment (FredrikAreschoug) [inline owldroid/src/main/java/com/hedvig/android/owldroid/feature/marketing/ui/MarketingFragment.kt]: style wise is this prefered? I have no real opinion about this but we should have the same settings

File: owldroid/src/main/res/layout/fragment_charity.xml (hunk 1 above)
Comment (hedvigoscar) [reply owldroid/src/main/res/layout/fragment_charity.xml]: My editor was misconfigured I think. I've fixed it to be standard android style now

File: owldroid/src/main/java/com/hedvig/android/owldroid/feature/marketing/ui/MarketingFragment.kt (hunk 2 above)
Comment (hedvigoscar) [reply owldroid/src/main/java/com/hedvig/android/owldroid/feature/marketing/ui/MarketingFragment.kt]: I think this is a mistake actually. I have no strong opinions other than that we should follow whatever `ktlint` produces. I'm a bit surprised that it didn't fix this

**Pull request merged**